release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1269,6 +1269,13 @@ Optionally, you can now override the default evaluation intervals for all polici
 
 For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc#ztp-configuring-pgt-compliance-eval-timeouts_ztp-advanced-policy-config[Configuring policy compliance evaluation timeouts for PolicyGenTemplate CRs].
 
+[id="ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries"]
+==== Configuring the hub cluster to use unauthenticated registries
+
+This release supports the use of unauthenticated registries when configuring the hub cluster. Registries that don't require authentication can be listed under `spec.unauthenticatedRegistries` in the `AgentServiceConfig` resource. 
+Any registry on this list is not required to have an entry in the pull secret validated by the `assisted-service`.
```

Comment:
I'm not clear on what is happening here:

> an entry in the pull secret validated by

Is the pull secret validated or the entry? Or, am I off here?

Reply:
Updated as suggested.